lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -32,6 +32,7 @@ import VitalSourceBookViewer from './VitalSourceBookViewer';
  * @typedef {'error-fetching'|
  *           'error-authorizing'|
  *           'error-reporting-submission'|
+ *           'blackboard_file_not_found_in_course'|
```

Comment:
The comment above this `@typedef` needs a slight revision. It currently mentions only `canvas_` and `error-`-prefixed values.